cypress/integration/submit_spec.js

```diff
@@ -50,6 +50,12 @@ describe('e2e test', () => {
         cy.get('#field-input--firstname').clear()
         cy.get('#field-input--lastname').click()
         cy.get('#field-error--firstname>span').should('contain', 'Please fill in your first name')
+        cy.get('#field-input--firstname').type('Test@')
```

Comment:
there shouldn't be back slash  \  in the first name error message according to the pattern